AC-Meeting-Guidelines.md

```diff
@@ -0,0 +1,38 @@
+# Architecture Committee Meeting Guidelines
+
+The Architecture Committee (AC) recommends following a set of guidelines for raising issues and adding topics to the meeting agenda.
+
+## Use a GitHub issue or an email to the mailing list first
+
+For a project like Kata Containers, with contributors spanning across so many different time zones, asynchronous discussions and reviews are the preferred communication media.
+
+Directly starting a proposal or trying to resolve new technical issues through the AC meeting has many drawbacks:
+1. It potentially leaves a large part of the community out of the initial dicussion.
+1. It does not give the AC meeting audience enough time to think through the proposal and give constructive and well thought feedback.
+1. It makes it hard to track and share the discussion outcome.
+
+As a consequence, the AC recommends using GitHub issues and PRs, or the kata-dev mailing list at first before adding new items to the AC meetings agenda, especially if you think the topic could be discussed and resolved in less than two weeks.
+
+## AC Agenda topics must be supported by a GitHub issue or PR, or an email sent to the mailing list
+
+Except for a few cases listed in the next section, any topic brought to the AC meeting must be linked to a GitHub issue, PR, or to an email to the mailing list.
+Unless the topic must be urgently discussed, the AC recommends waiting for one or two weeks before bringing the discussion to the AC meeting. That will give enough time for interested contributors to digest the issue or proposal before having a live discussion in the AC meeting.
+ 
+
+### Exceptions
+
+* Announcements
+* Presentations
+* Call for reviews of articles, blog posts, etc.
```

Comment:
I would add vulnerability disclosures to that list. And point to https://github.com/kata-containers/community/blob/master/VMT

Reply:
Added. Good point, thanks.

Reply:
I'm fine about having vulnerability disclosures added to that list, but we must be quite clear that the way to report those is via the VMT.

What I'm worried is that, unless well explained, people will prefer joining the call and exposing a security issue that should be "embargoed", rather than going through the official process.

Does my concern make sense?

Reply:
I'd suggest moving this section to the very top of the document and turning the VMT bullet into [an admonition](https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#warnings-and-other-admonitions) like this:

> **Important:**
>
> Do **not** follow this process if you think you have discovered a vulnerability. Instead, please use the [VMT](https://github.com/kata-containers/community/blob/master/VMT/VMT.md) process.

Reply:
> I'd suggest moving this section to the very top of the document and turning the VMT bullet into [an admonition](https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#warnings-and-other-admonitions) like this:
> 
> > **Important:**
> > Do **not** follow this process if you think you have discovered a vulnerability. Instead, please use the [VMT](https://github.com/kata-containers/community/blob/master/VMT/VMT.md) process.

@fidencio @jodh-intel Done, it is a top level admonition now.